depends/patches/boost/mingw32_process.patch

```diff
@@ -0,0 +1,21 @@
+Fix compiling for MinGW-w64 using std::filesystem 
+
+Upstream commit:
+ - https://github.com/boostorg/process/commit/a7b65bfc44f4af3a9d584fe9347fde0944c0167a
```

Comment:
This has been merged. Do we know which release it's in?

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/25696#issuecomment-1324936336).